### PR TITLE
Testcafe reporter with autogenerated client

### PR DESCRIPTION
**Pull request by kmalinin**

### CODE REVIEW

File: qase-testcafe/examples/.qaserc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "enabled": true,
     "logging": true,
-    "apiToken": "api_key",
-    "projectCode": "project_code",
+    "environmentId": 1,
```
Comment (nl-kaidzen) [inline qase-testcafe/examples/.qaserc:4]: Let's change back "api_key" and "project_code". We used these "keywords" at Readme.md

File: qase-testcafe/examples/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "example",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (nl-kaidzen) [inline qase-testcafe/examples/package.json:3]: -alpha.0

File: qase-testcafe/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-reporter-qase",
-  "version": "v1.1.1",
+  "version": "v1.2.0-alpha.4",
```
Comment (nl-kaidzen) [inline qase-testcafe/package.json:3]: Why minor and alpha.4?

**Review by nl-kaidzen — COMMENTED** [review]
Resolve issues from GithubActions with unused variables:
- endtime
-passed,
etc...

File: qase-testcafe/examples/.qaserc [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
     "enabled": true,
     "logging": true,
+    "environmentId": 1,
     "apiToken": "api_key",
-    "projectCode": "project_code",
+    "projectCode": "project_code"
```
Comment (kamilsk) [inline qase-testcafe/examples/.qaserc:6]: why did you drop comma?

**Review by kamilsk — APPROVED** [review]

**Review by nl-kaidzen — APPROVED** [review]

**Review by kamilsk — CHANGES_REQUESTED** [review]
so slow communication

**Review by kamilsk — APPROVED** [review]

**Pull request merged**